changelog/october2023/2023-10-27-load-balancers-added-load-balancer-is-now-available-in-.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Load Balancer is now available in WAW3 AZ
+status: added
+author:
+    fullname: 'Join the #load-balancer channel on Slack.'
+    url: 'https://slack.scaleway.com/'
+date: 10/27/2023
```

Comment:
need to set ISO format `yyyy-mm-dd`

Reply:
Fixed here: https://github.com/scaleway/docs-content/pull/2335/files#diff-0ea6b5184c67a3c58aa3f4d3699d244573f1c7c9cd1631dbe15788966e6ae23cR7